jazzygrid/patterns/home.php

```diff
@@ -0,0 +1,153 @@
+<?php
+/**
+ * Title: home
+ * Slug: jazzygrid/home
+ * Categories: hidden
+ * Inserter: no
+ */
+?>
+<!-- wp:cover {"dimRatio":0,"isUserOverlayColor":true,"minHeight":95,"minHeightUnit":"vh","isDark":false,"metadata":{"categories":["hidden"]},"style":{"spacing":{"padding":{"top":"1rem","right":"1rem","bottom":"1rem","left":"1rem"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-cover is-light" style="padding-top:1rem;padding-right:1rem;padding-bottom:1rem;padding-left:1rem;min-height:95vh"><span aria-hidden="true" class="wp-block-cover__background has-background-dim-0 has-background-dim"></span><div class="wp-block-cover__inner-container"><!-- wp:columns {"align":"wide","style":{"spacing":{"margin":{"top":"0px","bottom":"0px"},"blockGap":{"top":"0px","left":"0px"},"padding":{"top":"0rem","right":"0rem","bottom":"0rem","left":"0rem"}}}} -->
+<div class="wp-block-columns alignwide" style="margin-top:0px;margin-bottom:0px;padding-top:0rem;padding-right:0rem;padding-bottom:0rem;padding-left:0rem"><!-- wp:column {"verticalAlignment":"stretch","width":"25%","backgroundColor":"secondary"} -->
+<div class="wp-block-column is-vertically-aligned-stretch has-secondary-background-color has-background" style="flex-basis:25%"><!-- wp:cover {"url":"<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/beach-sea-coast-water-ocean-horizon.jpg","dimRatio":0,"minHeight":100,"minHeightUnit":"%","contentPosition":"bottom left","align":"full","style":{"spacing":{"blockGap":"0px","margin":{"top":"0px","bottom":"0px"},"padding":{"top":"1.5rem","right":"3rem","bottom":"1.5rem","left":"1.5rem"}},"color":[]},"layout":{"type":"default"}} -->
+<div class="wp-block-cover alignfull has-custom-content-position is-position-bottom-left" style="margin-top:0px;margin-bottom:0px;padding-top:1.5rem;padding-right:3rem;padding-bottom:1.5rem;padding-left:1.5rem;min-height:100%"><span aria-hidden="true" class="wp-block-cover__background has-background-dim-0 has-background-dim"></span><img class="wp-block-cover__image-background" alt="<?php echo __('', 'jazzygrid');?>" src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/beach-sea-coast-water-ocean-horizon.jpg" data-object-fit="cover"/><div class="wp-block-cover__inner-container"><!-- wp:site-logo {"width":120,"shouldSyncIcon":false,"align":"center"} /--></div></div>
```

Comment:
I don't think empty strings need to be translated. This looks as if it's intended to be a decorative image, so the empty alt should be fine.

```suggestion
<div class="wp-block-cover alignfull has-custom-content-position is-position-bottom-left" style="margin-top:0px;margin-bottom:0px;padding-top:1.5rem;padding-right:3rem;padding-bottom:1.5rem;padding-left:1.5rem;min-height:100%"><span aria-hidden="true" class="wp-block-cover__background has-background-dim-0 has-background-dim"></span><img class="wp-block-cover__image-background" alt="" src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/beach-sea-coast-water-ocean-horizon.jpg" data-object-fit="cover"/><div class="wp-block-cover__inner-container"><!-- wp:site-logo {"width":120,"shouldSyncIcon":false,"align":"center"} /--></div></div>
```